packages/backend/src/apps/paysg/actions/create-payment-form-submission-subscription/index.ts

```diff
@@ -0,0 +1,160 @@
+import type { IRawAction } from '@plumber/types'
+
+import { ZodError } from 'zod'
+import { fromZodError } from 'zod-validation-error'
+
+import StepError, { GenericSolution } from '@/errors/step'
+
+import { requestSchema } from './schema'
+
+const action: IRawAction = {
+  name: 'Create payment form submission for subscription',
+  key: 'createPaymentFormSubmissionSubscription',
+  description:
+    'Create a new submission for a payment form, and initiate a subscription',
+  arguments: [
+    {
+      label: 'Payment Form Link',
+      description: 'This can be found on your PaySG payment services dashboard',
+      key: 'formId',
+      type: 'string' as const,
+      required: true,
+      placeholder: 'e.g. https://pay.gov.sg/forms/aBC123Def456HijK789LMn',
+    },
+    {
+      label: 'FormSG Form ID',
+      description: 'Input the variable corresponding to form ID here',
+      key: 'formsg_form_id',
+      type: 'string' as const,
+      required: true,
+      variables: true,
+    },
+    {
+      label: 'FormSG Submission ID',
+      key: 'formsg_submission_id',
+      type: 'string' as const,
+      required: true,
+      variables: true,
+    },
+    {
+      label: 'FormSG Reference Field Answer',
+      key: 'nonce',
+      type: 'string' as const,
+      required: true,
+      variables: true,
+    },
+    {
+      label: 'Payer Name',
+      key: 'payer_name',
+      type: 'string' as const,
+      required: true,
+      variables: true,
+    },
+    {
+      label: 'Payer Email',
+      key: 'payer_email',
+      type: 'string' as const,
+      required: true,
+      variables: true,
+    },
+    {
+      label: 'Payer Address',
+      key: 'payer_address',
+      type: 'string' as const,
+      required: false,
+      variables: true,
+    },
+    {
+      label: 'Payer Identifier',
+      key: 'payer_identifier',
+      type: 'string' as const,
+      required: false,
+      variables: true,
+    },
+    {
+      label: 'Amount (in cents)',
+      key: 'amount_in_cents',
+      type: 'string' as const,
+      required: true,
+      variables: true,
+    },
+    {
+      label: 'Frequency',
+      key: 'frequency',
+      type: 'dropdown',
+      options: [
+        {
+          label: 'Monthly',
+          value: 'monthly',
+        },
+      ],
+      showOptionValue: false,
```

Comment:
```suggestion
      value: 'monthly',
```
suggest to add `value: 'monthly'` so that the dropdown will automatically be populated as there is only 1 option